var/spack/repos/builtin/packages/extrae/package.py

```diff
@@ -120,15 +131,20 @@ def configure_args(self):
             else ["--without-papi"]
         )
 
-        args += (
-            ["--with-dyninst=%s" % spec["dyninst"].prefix]
-            if spec.satisfies("+dyninst")
-            else ["--without-dyninst"]
-        )
+        if spec.satisfies("+dyninst"):
+            args += ["--with-dyninst=%s" % spec["dyninst"].prefix]
+
+            if spec.satisfies("@4.1.2:"):
+                args += [
+                    "--with-elfutils=%s" % spec["elfutils"].prefix,
+                    "--with-tbb=%s" % spec["tbb"].prefix,
```

Comment:
For future reference, we are converting the core to use f-strings (https://docs.python.org/3/tutorial/inputoutput.html) when we make updates to packages and are encouraging the same for packages.

Reply:
Updated.